test-data/unit/cmdline.test

```diff
@@ -581,7 +581,7 @@ m.py:6: error: Explicit "Any" is not allowed
 m.py:9: error: Explicit "Any" is not allowed
 
 [case testDisallowAnyExplicitVarDeclaration]
-# cmd: mypy m.py
+# cmd: mypy --python-version=3.6 m.py
```

Comment:
Similar for the Python version here. Maybe all these tests should be run with 3.6 unless they override it?